src/observers/form_link_interceptor.ts

```diff
@@ -38,6 +38,9 @@ export class FormLinkInterceptor implements LinkInterceptorDelegate {
     const method = link.getAttribute("data-turbo-method")
     if (method) form.setAttribute("method", method)
 
+    const turboAction = link.getAttribute("data-turbo-action")
+    if (turboAction) form.setAttribute("data-turbo-action", turboAction)
```

Comment:
Thank you for catching this missing attribute!

This method is becoming unwieldy. I'd considered an implementation that looped over the `<a>` element's attributes and mapped them to attributes on the `<form>` element, but decided against it.

As this list continues to grow, I wonder whether or not its time.

What do you think about implementing this method in terms of [Element.attributes](https://developer.mozilla.org/en-US/docs/Web/API/Element/attributes) and [Attr.localName](https://developer.mozilla.org/en-US/docs/Web/API/Attr/localName) and [Attr.value](https://developer.mozilla.org/en-US/docs/Web/API/Attr/value), special-casing for `[data-turbo-method]`?

```diff
diff --git a/src/observers/form_link_interceptor.ts b/src/observers/form_link_interceptor.ts
index 6df0203..a2a309f 100644
--- a/src/observers/form_link_interceptor.ts
+++ b/src/observers/form_link_interceptor.ts
@@ -35,14 +35,13 @@ export class FormLinkInterceptor implements LinkInterceptorDelegate {
     form.setAttribute("action", action)
     form.setAttribute("hidden", "")
 
-    const method = link.getAttribute("data-turbo-method")
-    if (method) form.setAttribute("method", method)
-
-    const turboConfirm = link.getAttribute("data-turbo-confirm")
-    if (turboConfirm) form.setAttribute("data-turbo-confirm", turboConfirm)
-
-    const turboStream = link.hasAttribute("data-turbo-stream")
-    if (turboStream) form.setAttribute("data-turbo-stream", "")
+    for (const { localName, value } of link.attributes) {
+      if (localName == "data-turbo-method") {
+        form.setAttribute("method", value)
+      } else if (localName.startsWith("data-turbo")) {
+        form.setAttribute(localName, value)
+      }
+    }
 
     this.delegate.formLinkClickIntercepted(link, form)
 ```

Reply:
That works too! Let's address the issue then we can assess after the fact